apps/communications/dialer/js/mmi.js

```diff
@@ -30,6 +30,8 @@ var MmiManager = {
       return Promise.resolve();
```

Comment:
This was a really awkward way to do this, but let's not modify it in the scope of this patch.